### ISSUE TITLE
Problem with the default atom autocomplete behaviour

### ISSUE THREAD

**Issue by mrkaspa** [msaraiva/atom-elixir#30]
It doesn't autocomplete with variables on the module, functions on the same module or with previous used words in the files like it's the common atom behaviour

**Comment by msaraiva** [comment]
@mrkaspa Autocomplete for used words in the file is not supported yet. Variables on module, functions on the same module should all work as long as the module has been successfully compiled. There's a new version v0.2.1. You might try it and see if the problem is gone. Let me know if it's not.

**Comment by styx** [comment]
I also have a problem with autocomplete. you_complete_me stops working after enabling atom-elixir plugin. Don't know how that should be implemented, but it seems to be possible to make them work together. If I'm not missing typescript plugin worked well with ycm.

**Comment by mrkaspa** [comment]
@msaraiva yeah this has to be posible and it's pretty useful, I've got the lastest version but doesn't work yet

**Comment by msaraiva** [comment]
I guess [this](https://github.com/msaraiva/atom-elixir/issues/36) would solve the issue. I'll try to implement this before the next release. PRs are welcome.

Thanks for reporting!

**Comment by mrkaspa** [comment]
@msaraiva you were right https://github.com/msaraiva/atom-elixir/pull/37

**Comment by msaraiva** [comment]
@mrkaspa yeah, but we need to have that value based on a preference so the user can choose the preferable behaviour.

**Comment by rubencaro** [comment]
I guess we all waiting for #36
